cmd/fetch.go

```diff
@@ -13,13 +13,14 @@ import (
 func NewFetchCommand(f Factory, ioStreams ioes.IOStreams) *cobra.Command {
 	o := &FetchOptions{IOStreams: ioStreams}
 	cmd := &cobra.Command{
-		Use:     "fetch",
-		Short:   "fetch logbook info for a dataset reference",
+		Use:     "fetch DATASET [DATASET...]",
+		Short:   "fetch logbook info for datasets",
```

Comment:
I think this is actually fetching the logs, not the logbook. Is that right and I should change the description?

(Along those lines, is there a meaningful distinction between this and `qri log` other than this operates on remote information and that [only?] operates on the local repo?)

Reply:
Sorta correct. it's fetching logbook data, just not _persisting_ it.

 As we discussed on discord, We'd like to merge `fetch` into the `log` command. Given how long that might take, I think it's still worth updating the doc here